motion/lib/src/motion_builder.dart

```diff
@@ -82,8 +82,11 @@ class _MotionStreamBuilderState extends State<MotionStreamBuilder> {
 
   /// The effective filter quality. It ensures that the platform is not Safari Mobile, whose implementation
   /// of Transform.filterQuality renders artifacts.
-  FilterQuality? get filterQuality =>
-      Motion.instance.isSafariMobile ? null : widget.filterQuality;
+  FilterQuality? get filterQuality => Motion.instance.isSafariMobile
+      ? null
+      : kIsWeb && widget.filterQuality == FilterQuality.high
```

Comment:
This check should be performed by the user, as different versions of Flutter may or may not cause the problem you are attempting to fix. I believe should not be endorsed by the plugin and the choice should be left to the developers who will pass the appropriate `filterQuality`.

Reply:
Agree with you but this package doesn't work on web which led me to these changes.

Reply:
I understand it causes issues on the Web, but the app/package using **motion** should perform the check to `kIsWeb` themselves, instead of being forced to use an arbitrary value.

This is because when the Flutter team will have fixed this `FilterQuality` issue, some developers may want to use the filter quality that no longer causes the bug, without waiting for a package update.

Reply:
I got your point. I can remove this from the PR or you can also update your package to support auto initialisation.